pyro/distributions/householder.py

```diff
@@ -57,6 +57,7 @@ class HouseholderFlow(TransformModule):
     codomain = constraints.real
     bijective = True
     event_dim = 1
+    volume_preserving = True
```

Comment:
is `volume_preserving` used anywhere?

Reply:
Not yet, but I think it's useful in case we want to incorporate it into the tests in the future. For instance, I use a similar `autoregressive = True` flag in other flows so I know how to run the Jacobian test